x/govshuttle/keeper/msg_server_test.go

```diff
@@ -86,6 +86,23 @@ func (suite *KeeperTestSuite) TestMsgExecutionByProposal() {
 			func(uint64, sdk.Msg) {},
 			true,
 		},
+		{
+			"fail - MsgLendingMarketProposal - validate basic logic",
```

Comment:
It seems that the reason this test case is failing is not related to validate basic. Could you please check it?

Reply:
I modified test logic